.gitlab-ci.yml

```diff
@@ -40,7 +45,18 @@ deploy_to_sonatype:
     - set -x
 
     - echo "Building release..."
-    - mvn -DperformRelease=true --settings ./settings.xml clean validate jar:jar source:jar javadoc:jar assembly:single deploy
+    - mvn -DperformRelease=true --settings ./settings.xml clean verify deploy
+
+  artifacts:
+    expire_in: 12 mos
```

Comment:
Saving this key artifact isn't really strictly needed as we push the key to SSM and the ubuntu keyservers but it's valuable here to have as a way to compare artifacts if needed (or to push that key to a different set of keyservers) for the duration of validity. The public key is also about 2kb and the job is run once every year so I don't believe that it will cause any issues with space utilization.